functests/tests.sh

```diff
@@ -104,6 +104,66 @@ function do_upgrade_test() {
     uninstall_directcsi
 }
 
+function do_version_adapter_test() {
+    echo "One"
+    wget --quiet --output-document="kubectl-direct_csi_$1" "https://github.com/minio/direct-csi/releases/download/v$1/kubectl-direct_csi_$1_linux_amd64"
+    chmod a+x "kubectl-direct_csi_$1"
```

Comment:
We don't need to `wget` again as the `kubectl-direct_csi_1.4.3` and `kubectl-direct_csi_1.3.6` are already downloaded in the previous `upgrade_tests` run.